libs/e2e/src/index.ts

```diff
@@ -6,6 +6,7 @@ export * from "./lib/describer/modelAssessment/causalAnalysis/describeCausalAnal
 export * from "./lib/describer/modelAssessment/dataExplorer/describeDatasetExplorer";
 export * from "./lib/describer/modelAssessment/errorAnalysis/describeErrorAnalysis";
 export * from "./lib/describer/modelAssessment/featureImportances/individualFeatureImportance/describeIndividualFeatureImportance";
-export * from "./lib/describer/modelAssessment/modelStatistics/describeModelStatistics";
+export * from "./lib/describer/modelAssessment/modelOverview/describeModelOverview";
```

Comment:
just a fyi, this has to be updated in AML once we release and this change is being used in AML.

Reply:
Good point! Need to keep that on our radar for when we do that. Or do you think I should proactively go and upgrade the version Studio uses and make this change?

Reply:
You can update this once we release new version with this code change, that will be fine.